reana_client/cli/files.py

```diff
@@ -31,6 +31,9 @@
 from reana_commons.utils import click_table_printer
 
 
+FILES_BLACKLIST = ('.git/', '/.git/')
```

Comment:
Is there a need for `'/.git/'` ?

Reply:
Yes,  this is how `reana-client du` prints the information. This is what `reana-client` obtains from the server:

```python
...
{'name': '/.git/info/exclude', 'size': '4.0K'}
{'name': '/.git/info', 'size': '4.0K'}        
{'name': '/.git/branches', 'size': '0'}       
{'name': '/.git', 'size': '180K'}             
...
```